tractor/_trionics.py

```diff
@@ -111,11 +111,16 @@ async def run_in_actor(
         the actor is terminated.
         """
         mod_path = fn.__module__
+
+        if name is None:
+            # use the explicit function name if not provided
+            name = fn.__name__
+
         portal = await self.start_actor(
             name,
             rpc_module_paths=[mod_path] + (rpc_module_paths or []),
             bind_addr=bind_addr,
-            statespace=statespace,
+            # statespace=statespace,
```

Comment:
Are you wanting to leave these commented lines

Reply:
nice catch 😏